modules/express/test/index.test.ts

```diff
@@ -116,7 +95,27 @@ const fg = {
   customCommands: {}
 };
 
-type MyTypeMap = TypeMapFromFunctionalGroup<typeof fg>;
+interface MyTypeMap extends GeneralTypeMap {
+  entities: {
+    member: {
+      request: Member;
+      response: MemberResponse;
+    };
+    diary: {
```

Comment:
[imo] since diary has the same type for Request and Response, this part can be shortened by using `ReqRes`, as suggested above